src/tools/installers/CURRENT_VERSION

```diff
@@ -5,3 +5,5 @@ VDO_VERSION               = 8.4.0.0
 VDO_MARKETING_VERSION     = 8.4
 VDO_PROJECT_CODENAME      = "Potassium"
 VDO_MODNAME               = kvdo
+VDO_LINUXNEXT_MODNAME     = dm-vdo
+VDO_LINUXNEXT_MODVER      = 8.3
```

Comment:
Upstream, there is no meaningful version for dm-vdo (apart for the kernel version) so don't define it here.For the moudle name, it should clearly be VDO_UPSTREAM_MODNAME. I'd rather not define it at all in this file,  but I guess we need to put it somewhere.